vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/model/implementation/Serializer.java

```diff
@@ -201,8 +221,26 @@ public static String getConstantName(LanguageStringConstant languageStringConsta
 
 	/**
 	 * Creates a String representation corresponding to the name of a given
-	 * {@link DatatypeConstant}. Example: ""c"^^<http://example.org/mystring>"
+	 * {@link DatatypeConstant} without an IRI.
+	 * 
+	 * @see <"https://github.com/knowsys/vlog4j/wiki"> for wiki.
+	 * @param datatypeConstant a {@link DatatypeConstant}
+	 * @return String representation corresponding to a given
+	 *         {@link DatatypeConstant}.
+	 */
+	public static String getShortConstantName(DatatypeConstant datatypeConstant) {
+		if (datatypeConstant.getDatatype().equals(PrefixDeclarations.XSD_STRING)) {
+			return "\"" + datatypeConstant.getLexicalValue() + "\"";
+		} else {
+			return datatypeConstant.getLexicalValue();
+		}
```

Comment:
This should also handle:
  - LanguageStrings, which should serialise to `"string"@en`,
  - `xsd:integer`, which should serialise to `23`,
  - `xsd:decimal`, which should serialise to `1234.5678`, and
  - `xsd:double`, which should serialise to something like `12.345E67`.
Note that any of the numbers might be negative, and the exponent in doubles might have a different sign than the mantissa.